Sources/FunctionalTableData/TableView/FunctionalTableData+UITableViewDataSource.swift

```diff
@@ -57,4 +57,347 @@ extension FunctionalTableData {
 			return cellConfig.actions.hasEditActions || self.tableView(tableView, canMoveRowAt: indexPath) || cellConfig.style?.selected != nil
 		}
 	}
+	
+	class OldFunctionalTableDataImpl: FunctionalTableDataImpl {
```

Comment:
Styling: Can we move some of these types to their own files to short this file up a bit?